locustempus/main/tests/test_views.py

```diff
@@ -1086,7 +1080,7 @@ def test_list_project_faculty(self):
             )
         )
         response = self.client.get(
-            "/course/{}/project/".format(self.sandbox_course.pk))
+            "/course/{}/".format(self.sandbox_course.pk))
```

Comment:
It's better practice to use `reverse` to construct URLs in tests, like this: https://github.com/ccnmtl/mediathread/blob/master/mediathread/main/tests/test_views.py#L1619

Then you don't have to update everything here if the URLs change.

Reply:
Great point, I've replaced all the hard coded URLs.